src/rules/function-url-quotes/__tests__/index.js

```diff
@@ -145,6 +145,10 @@ testRule(rule, {
     code: "@font-face { font-family: 'foo'; src: url($variable + \"foo.ttf\" + $variable); }",
   }, {
     code: "a { background: url(\"/images/my_image@2x.png\") }",
+  }, {
+    code: "@-moz-document url-prefix() {}",
+  }, {
+    code: "a { background: url() }",
```

Comment:
As I get, `url()` without quotes should reject because **it has empty string as argument**.
Maybe to create parameter `no-quotes-for-empty`?